var/spack/repos/builtin/packages/py-versioneer/package.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyVersioneer(PythonPackage):
+    """Versioneer is a tool to automatically update version strings (in
+    setup.py and the conventional ‘from PROJECT import _version’ pattern) by
+    asking your version-control system about the current tree."""
+
+    homepage = "https://github.com/warner/python-versioneer"
+    url      = "https://github.com/warner/python-versioneer/archive/0.18.tar.gz"
+    git      = "https://github.com/warner/python-versioneer.git"
+
+    maintainers = ['scemama', 'warner']
+
+    version('0.18', sha256='cf895b67f5bc62d61c4837458069ded8f66b4e5764c19f7253c51ee27e8b3a99')
+
+    depends_on('python@2.7:')
+#  depends_on('python@2.7:2.8.999', type=('build', 'run'), when='@0.10:1.999')
+#  depends_on('python@3:', type=('build', 'run'), when='@0.18:')
```

Comment:
I would remove these lines. It looks like this package supports Python 2.6+ and 3.2+, but there's no restriction preventing it from being installed with older versions. If you do want to keep it, it should look like:
```python
depends_on('python@2.6:2.8,3.2:', type=('build', 'run')
```

Reply:
I had a weird error with Travis, so I was testing different Python versions. I guess the Travis errors came from the unicode in the docstring that you mentioned, so we will see soon.

Reply:
Yep, looks like it was the docstring.

Reply:
The above comment is still valid, either remove the line or make it more specific with the right deptype